packages/core/strapi/lib/services/server/register-routes.js

```diff
@@ -40,7 +41,7 @@ const registerAdminRoutes = (strapi) => {
 
   strapi.server.routes({
     type: 'admin',
-    prefix: '/admin',
+    prefix: `${prefix}/admin`,
```

Comment:
Hi derrick.
I made this change to be able to deploy my infrastructure in kubernetes, do you think there is a chance of passing and entering a next version?

Reply:
Probably not in the next version since we released that today. You should be able to use patch-package to apply the patch to your own project until we can merge and release.

Reply:
Already do this in my project.
but considering the changes do you think it is possible to enter the next versions?